pkgs/by-name/um/umockdev/package.nix

```diff
@@ -15,11 +15,17 @@
   python3,
   replaceVars,
   systemdMinimal,
+  udev,
   usbutils,
   vala,
   which,
+  withSystemd ? udev.passthru.features.withSystemd,
```

Comment:
I don't think this should be an argument to the package. I think it's a matter of fact whether this works with the `libudev` implementation or not, and there's no reason to allow overrides to change that.